src/pages/community/index.md

```diff
@@ -38,7 +38,7 @@ Twitter: [@katacontainers](https://twitter.com/katacontainers)
 ## Meetings
 
 **Architecture Committee**\
-The Architecture Committee works on the technical direction of the project. Meetings are on Tuesdays at 1500 UTC. You can [add this meeting](https://zoom.us/meeting/tJUpc-2opjkjGNCIo1YFpKlcoLn909gWv_iH/ics?icsToken=98tyKuCpqj8uGtyXuRuDRowcBo-gLPTxiHpEjY1-tRC0OgZDSQ_1Oc5Ma-ImF-_G) to your calendar and find meeting agendas & call info [here](https://etherpad.opendev.org/p/Kata_Containers_2021_Architecture_Committee_Mtgs). 
+The Architecture Committee works on the technical direction of the project. Meetings are on Tuesdays at 1500 UTC. You can [add this meeting](https://zoom.us/meeting/tJUpc-2opjkjGNCIo1YFpKlcoLn909gWv_iH/ics?icsToken=98tyKuCpqj8uGtyXuRuDRowcBo-gLPTxiHpEjY1-tRC0OgZDSQ_1Oc5Ma-ImF-_G) to your calendar and find meeting agendas & call info [here](https://etherpad.opendev.org/p/Kata_Containers_2022_Architecture_Committee_Mtgs).
```

Comment:
How about we have a etherpad with this URL and we embed that link into this file:

https://etherpad.opendev.org/p/Kata_Containers_Architecture_Committee_Mtgs

We could then have links in that file to each years actual etherpad.

Reply:
Another choice is just to use the URL without the year for the current year, at the end of this year/start of next year rename it to that year, and create a new blank page for the new year.

When the year changes, only renaming the etherpad is required.

Reply:
Ack. Either option wfm ;)

Reply:
@jodh-intel  Updaed.